server/api/appointment_handler.py

```diff
@@ -77,5 +78,31 @@ def get_appointment(uid):
         .filter(EventTypes.user_id == uid).all()
     return jsonify([appointments.to_dict() for appointments in all_appointments])
 
+@appointment_handler.route('/appointment/<int:id>', methods=['DELETE'])
+@check_token
+def delete_appointment(id):
+    try:
+        user = Users.query.get(int(request.get_json().get("user_id")))
+        appointment_to_delete = Appointments.query.get(id)
+
+        if appointment_to_delete is None:
+            raise Exception("Appointment to delete not found")
+        
+        google_client = GoogleClient(access_token=user.access_token, refresh_token=user.refresh_token)
+        google_client.delete_calendar_event(appointment_to_delete.google_event_id)
+
+        db.session.delete(appointment_to_delete)   
+        db.session.commit()
+
+        return jsonify({
+            "success":True,
+            "response":"Appointment deleted"
+        }) 
+    except Exception as e:
+        return jsonify({
+            "success":False,
+            "response":e
+        })
+
```

Comment:
There is possibility that the error occur in `db.session.commit()` and I my point of view, returning the `Exception` derectly to the frontend is kinda dangerous, because user may get the error information and find a way to hack the server.
Maybe it is better to change the `"response":e` as a static preset string